src/js/core.js

```diff
@@ -356,21 +356,20 @@ function MediumEditor(elements, options) {
         }
     }
 
-    function initPasteHandler() {
-        var pasteOptions = Util.extend(
-            {},
-            this.options.paste,
-            // Backwards compatability
-            {
-                forcePlainText: this.options.forcePlainText, // deprecated
-                cleanPastedHTML: this.options.cleanPastedHTML, // deprecated
-                disableReturn: this.options.disableReturn,
-                targetBlank: this.options.targetBlank,
-                contentWindow: this.options.contentWindow,
-                ownerDocument: this.options.ownerDocument
-        });
-
-        this.pasteHandler = new PasteHandler(this, pasteOptions);
+    function initPasteHandler(options) {
+        // Backwards compatability
+        var defaultsBC = {
+            forcePlainText: this.options.forcePlainText, // deprecated
```

Comment:
I had a fancy mixDefaults thing I was working on, that simply detected the use of a non-default deprecated arg and passed along the right value to the right place ... it's why I added getObject and setObject to Util ... i've been trying to get this same patch going for some time now, but ended up breaking a buch of tests because of half-implementation ... (my mixDefaults was doing the initCommands work too, but that change requires Button to be an extension, too)